spec/features/brexit_checker/email_signup_spec.rb

```diff
@@ -40,8 +40,8 @@
     before do
```

Comment:
We should stick a note in these for the testing discussion,
Turns out [we should likely be using Climate Control here](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#testing-utilities)